ga4gh/sqliteBackend.py

```diff
@@ -0,0 +1,112 @@
+"""
+Library for accessing SQLite-backed data
+optimized for typical REST queries as defined by the GA4GH server.
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import sqlite3
+
+
+def sqliteRows2dicts(sqliteRows):
+    """
+    unpacks sqlite rows as returned by fetchall
+    into an array of simple dicts.
+    :param sqliteRows: array of rows returned from fetchall DB call
+    :return:  array of dicts, keyed by the column names.
+    """
+    return map(lambda r: dict(zip(r.keys(), r)), sqliteRows)
+
+
+def sqliteRow2Dict(sqliteRow):
+    """
+    As above, but for a single row
```

Comment:
Either put in a full description or add the function name. (Functions might move around in future)